### PR TITLE
Staking copy fixes en

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
## Related Issue

Resolve

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -8933,7 +8933,7 @@ export default defineMessages({
     TR_STAKE_UNSTAKING_TAKES: {
         id: 'TR_STAKE_UNSTAKING_TAKES',
         defaultMessage:
-            'Unstaking currently takes {count, plural, one {# day} few {# days} other {# days}}. Once completed, you can trade or send it.',
+            'Unstaking currently takes {count, plural, one {# day} other {# days}}. Once completed, you can trade or send it.',
```
Comment (tomasklim) [inline packages/suite/src/support/messages.ts:8936]: few should be in all strings
because 1 den, 2 dny, 5 dnů

Comment (MiroslavProchazka) [reply packages/suite/src/support/messages.ts:8936]: It was not working with "few" parameter, Crowdin showed syntax error. Thats why we get rid of it

Comment (tomasklim) [reply packages/suite/src/support/messages.ts:8936]: but it is valid for Czech, so it should me modified there

Comment (MiroslavProchazka) [reply packages/suite/src/support/messages.ts:8936]: We will take care of it in Crowdin as well

**Review by tomasklim — APPROVED** [review]

**Pull request merged**